src/main/java/logisticspipes/pipes/PipeFluidSatellite.java

```diff
@@ -243,10 +238,10 @@ protected void ensureAllSatelliteStatus() {
         if (MainProxy.isClient()) {
             return;
         }
-        if (satelliteId == 0 && PipeFluidSatellite.AllSatellites.contains(this)) {
```

Comment:
why did you delete this check?

Reply:
you don't need to check if an item is in a collection before removing it, same with adding